.gitignore

```diff
@@ -29,6 +29,9 @@ output*.log
 loop
 !cmd/loop/
 
+loopd
```

Comment:
What command do you use to build?
None of the Makefile commands nor `go build ./...` / `go install ./...` actually seem to create these files, that's why we removed the lines in #115.
Not a big deal to add them back, just curious if they are indeed still needed.

Reply:
Yeah, this is wrong, since it will ignore everything in the `loopd` folder. 

> None of the Makefile commands nor `go build ./...` or `go install ./...` actually seem to create these files, that's why we removed the lines in #115.

I'm also didn't see the binary appear again, maybe you just have it lying around?

Reply:
I run `go build` inside `cmd/loopd` .

For me, it does not ignore everything in the loopd. If I create a file there, it is picked up by git. Are you sure it does not work for you?

Reply:
Other question is why the same lines for `loop` are still in the file if they cause everything to be ignored.

Reply:
Ah, yes, if you go into a directory with a main package then the binary is indeed created there.
I think what we really should do is add the `make build` and `make install` commands and then add that generated binary to the `.gitignore`.

There is a new top-level `loopd` directory (not `cmd/loopd`) that would be ignored/untracked with these changes.

Reply:
Ah, there is a `loopd` directory now. Ok, removed the .gitignore change. Alternatively we could also have added `!loopd` to .gitignore I think. How do you usually run locally then if it isn't with `go build`?

Reply:
Yes, that was Johan's suggestion before we decided to remove it altogether.
I always run `go build ./...` in the root directory to check for compilation errors. Or `go install ./...` if I want to update the binaries in my `$GOPATH/bin` directory.

Reply:
Hmm, installing the binaries for local dev isn't something I normally do. Sometimes I also have multiple versions.